### PR TITLE
[develop] Crypsis : You can only insert one map per page #2817

### PR DESCRIPTION
**Pull request by xillibit**

### CODE REVIEW

**Comment by rich20** [comment]
- [x] Success

**Comment by 810** [comment]
- [ ] Failed: Unable to display Google Map.

If you insert 2 maps in 1 message. Both locations are loaded in the first message

**Comment by rich20** [comment]
Sorry, you're right. I've tested only with one map per post.

File: components/com_kunena/site/template/crypsis/layouts/bbcode/map/default.php [hunk 1]
```diff
@@ -24,7 +24,7 @@
 	$id = 0;
```
Comment (810) [inline components/com_kunena/site/template/crypsis/layouts/bbcode/map/default.php]: Please update this to (New look/api, code still work): 
$this->addScript('http://maps.google.com/maps/api/js?v=3.exp&signed_in=true');
$id = 0;

**Comment by 810** [comment]
When i do
```` ->set('mapid', ++ $bbcode->parent->id)````
The maps are loaded correct, but the action buttons are gone also moderate is gone

**Comment by xillibit** [comment]
You can try now, i have updated it

**Comment by 810** [comment]
Code work, but please fix codestyle for : var $mapid=0; and $this->mapid++;

**Comment by xillibit** [comment]
Done

File: libraries/kunena/bbcode/bbcode.php [hunk 2]
```diff
@@ -328,6 +328,9 @@ public function getEbayItemFromCache($ItemID)
  */
 class KunenaBbcodeLibrary extends BBCodeLibrary {
 	var $default_smileys = array();
+
+	public $mapid=0;
```
Comment (810) [inline libraries/kunena/bbcode/bbcode.php]: ```` public $mapid = 0; ````

**Comment by xillibit** [comment]
Corrected

**Comment by 810** [comment]
:+1:

**Pull request merged**